index.php

```diff
@@ -183,8 +191,15 @@ function gutenberg_edit_site_export_theme_create_zip( $filename, $theme ) {
 	// Load templates into the zip file.
 	$templates = gutenberg_get_block_templates();
 	foreach ( $templates as $template ) {
-		$template->content = _remove_theme_attribute_from_content( $template->content );
 
+		//Currently, when building against CHILD themes of Blockbase, block templates provided by Blockbase, not modified by the child theme or the user are included in the page. This is a bug.
+		//if the theme is blockbase and the source is "theme" we don't want it
+		if ($template->source === 'theme' && $template->theme === 'blockbase') {
```

Comment:
this will fail if the theme is in a subdirectory. I'm getting this when I dump `$template->theme`:

` public 'theme' => string 'a8c-themes/blockbase'`

Reply:
Another good catch.  Refactored the check to `... && strpos($template_part->theme, 'blockbase') !== false` to account for that.